docs/usage.md

```diff
@@ -1,89 +1,118 @@
+You can ignore analysis results for certain files for example 3rd party modules.
+For that use the -i parameter of the analyze command:
+ -i SKIPFILE, --ignore SKIPFILE, --skip SKIPFILE
+                        Path to the Skipfile dictating which project files
+                        should be omitted from analysis. Please consult the
+                        User guide on how a Skipfile should be laid out.
 
-Large projects should use postgresql for performance reasons.
+For the skip file format see the [user guide](https://github.com/Ericsson/codechecker/blob/master/docs/user_guide.md#skip-file ).
 
 ```
-CodeChecker check -n myproject_v1 –postgresql -b make
-CodeChecker check -n myproject_v2 –postgresql -b make
-
-# Runs analysis, assumes that postgres is available on the default 5432 TCP port, 
-# codechecker db user exists and codechecker db can be created.
-# Please note that this use case is also supported with SQLITE db.
+ CodeChecker analyze -b "make" -i ./skip.file" -o ./reports
 ```
 
-Start the webserver and view the diff between the two results in the web browser
-```
-CodeChecker server –postgresql
+### Enable/Disable Checkers
+Clang and Clang-tidy and Ericsson internal clang checkers are also devivered with the codechecker package.
+
+You can list the checkers using the following command
+ CodeChecker checkers --details
+those marked with (+) are enabled by default.
 
-firefox localhost:8001
+You may want to enable more checkers or disable some of them using the -e, -d switches of the analyze command.
+
+For example to enable alpha checkers additionally to the defaults
 ```
-Start the webserver and view the diff between the two results in command line
+ CodeChecker analyze -e alpha  -b "make" -i ./skip.file" -o ./reports
 ```
-CodeChecker cmd diff –b myproject_v1 –n myproject_v2 –p 8001 -new
 
-# Assumes that the server is started on port 8001
-# then shows the new bugs in myproject_v2 compared to baseline myproject_v1.
+### Identify files that were failed to analyze
+After execution of
+ CodeChecker analyze build.json -o reports
+the failed analysis output is collected into 
+ ./reports/failed
+directory.
 
-CodeChecker cmd diff –b myproject_v1 –n myproject_v2 –p 8001 -unresolved
+This means that analysis of these files failed and there is no clang static analyzer output for these compilation commands.
 
-# Assumes that the server is started on port 8001
-# then shows the unresolved bugs in myproject_v2 compared to baseline myproject_v1.
+### Showing new bugs in your code after local edit
+Say that you made some local changes in your code (tmux in our example) and you wonder whether you introduced any new bugs.
 
-CodeChecker cmd diff –b myproject_v1 –n myproject_v2 –p 8001 -resolved
+Let's assume that you already stored the tmux analysis results in a CodeChecker server running on localhost:8555 under run name tmux See Step3: CodeChecker store ./reports --host localhost --port 8555 --name tmux
```

Comment:
URLs, filenames, identifiers should be formatted as code, between backticks.

The command too.